frontend/src/components/template/TheNavBar.vue

```diff
@@ -5,14 +5,14 @@
         <li id="home-link" :class="tabClass($route, 'Home')">
           <router-link @click="resetAlert" :to="{ name: 'Home'}">Home</router-link>
         </li>
-        <li id="eligibility-link" :class="menuTabClass($route, '/eligibility')">
+        <li id="eligibility-link" :class="menuTabClass($route, '/eligibility')" v-if="hasEligibilityPermission()">
           <div class="dropdown" :key="eligibilityDropdownKey" v-on:click="this.refreshEligibility()">
```

Comment:
Not related to changes in this PR but this file has `refreshEligibilty` and `refreshCoverageEnrollment` defined as method.
and `refreshEligibility` and `refreshEnrollment` actually used in the template.
Any idea what's going on here and what they're doing?

Reply:
These are used to force the menu to refresh with the correctly selected option when the tab contents are changed. Because of how the router-view is rendered these aren't updated by default and may will show the old option. The latter method naming is a bug that I will fix.

I actually took a closer look at this and it might not be required anymore. I'll do some more testing and remove in a future branch if I can.